src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -39,12 +39,14 @@ if NOT [%ERRORLEVEL%]==[0] (
     echo Warning: An error occurred while configuration https port.
   )
 
+REM
 REM Start the self hosted WCF Test Service
-call %~dp0..\..\..\..\bin\Wcf\tools\SelfHostedWcfService\SelfHostedWcfService.exe %*
+REM
+call %~dp0..\..\..\..\bin\Wcf\tools\SelfHostedWcfService\SelfHostedWcfService.exe
```

Comment:
Question: Will this script pause here and continue to run when SelfHostedWcfService process either exits gracefully (by pressing enter) or is killed by taskkill?